packages/terraform.rb

```diff
@@ -3,20 +3,20 @@
 class Terraform < Package
   description 'Terraform is a tool for building, changing, and combining infrastructure safely and efficiently.'
   homepage 'https://www.terraform.io/'
-  version '1.3.8'
+  version '1.3.9'
   license 'Apache-2.0, BSD-2, BSD-4, ECL-2.0, imagemagick, ISC, JSON, MIT, MIT-with-advertising, MPL-2.0 and unicode'
   compatibility 'all'
   source_url({
-    aarch64: "https://releases.hashicorp.com/terraform/#{version}/terraform_#{version}_linux_arm.zip",
+    aarch64: "https://releases.hashicorp.com/terraform/#{version}/terraform_#{version}_linux_arm64.zip",
```

Comment:
I suspect this won't work, as arm64 Chromebooks use a arm64 kernel, but still use an armv7l userland.

Reply:
I never even considered that.  Let me fix that.